automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/notebooks/NotebookRKernelSpec.scala

```diff
@@ -74,7 +74,7 @@ class NotebookRKernelSpec extends RuntimeFixtureSpec with NotebookTestUtils {
           val output = notebookPage.executeCell("""install.packages("httr")""", installTimeout)
           output shouldBe 'defined
           output.get should include("Installing package into")
-          output.get should include("/home/jupyter-user/.rpackages")
```

Comment:
these fixes (replacing `/home/jupyter-user/.rpackages` with `/usr/local/lib/R/site-library`) are temporary. after jenkins set up the next auto created PR (Custom image hash update) then I will be changing this string to `/home/jupyter-user/notebooks/packages`. We configure R to install to this new directory in the leo changes for persisting user installed packages